src/eva/plotting/emcpy/plot_tools/figure_driver.py

```diff
@@ -65,9 +65,21 @@ def figure_driver(config, data_collections, timing, logger):
         if batch_conf:
             # Get potential variables
             variables = batch_conf.get('variables', [])
+
             # Get list of channels
             channels_str_or_list = batch_conf.get('channels', [])
             channels = parse_channel_list(channels_str_or_list, logger)
+            step_vars = channels
+            step_var_name = 'channel'
+            title_fill = ' Ch. '
+
+            # Get list of levels
+            levels_str_or_list = batch_conf.get('levels', [])
+            levels = parse_channel_list(levels_str_or_list, logger)
+            if levels != []:
```

Comment:
I believe the more "pythonic" way of addressing this conditional would be:

`if levels:`

See here: https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty

Reply:
Good to know, thanks.  Change made.